ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,606 @@
+], 'Bleed');
+
+export const MeleeAttackSpeedDebuff = withLabel(
+	makeTristateDebuffInput(ActionId.fromSpellId(6343), ActionId.fromSpellId(12666), 'thunderClap'),
+	'Thunder Clap',
+);
+
+export const MeleeHitDebuff = withLabel(
+	makeBooleanDebuffInput({actionId: ActionId.fromSpellId(24977), fieldName: 'insectSwarm'}),
+	'Insect Swarm',
+);
+
+// TODO: Classic
+export const SpellISBDebuff = withLabel(
+	makeBooleanDebuffInput({actionId: ActionId.fromSpellId(17803), fieldName: 'improvedShadowBolt'}),
+	'Improved Shadow Bolt',
+);
+
+export const SpellScorchDebuff = withLabel(
+	makeBooleanDebuffInput({actionId: ActionId.fromSpellId(12873), fieldName: 'improvedScorch', minLevel: 40}),
+	'Improved Scorch',
+);
+
+export const SpellWintersChillDebuff = withLabel(
+	makeBooleanDebuffInput({actionId: ActionId.fromSpellId(28595), fieldName: 'wintersChill', minLevel: 40}),
+	'Winters Chill',
+);
+
+// TODO: Classic
+// export const SpellDamageDebuff = InputHelpers.makeMultiIconInput([
+// 	makeBooleanDebuffInput(ActionId.fromSpellId(47865), 'curseOfElements'),
+// ], 'Spell Dmg');
+
+// TODO: Classic
+export const HuntersMark = withLabel(
+	makeTristateDebuffInput(ActionId.fromSpellId(14325), ActionId.fromSpellId(19425), 'huntersMark'),
+	`Hunter's Mark`,
+);
+export const JudgementOfWisdom = withLabel(
+	makeBooleanDebuffInput({actionId: ActionId.fromSpellId(20355), fieldName: 'judgementOfWisdom', minLevel: 38}),
+	'Judgement of Wisdom',
+);
+
+// Misc Debuffs
+export const JudgementOfLight = makeBooleanDebuffInput({actionId: ActionId.fromSpellId(20346), fieldName: 'judgementOfLight', minLevel: 30});
+export const CurseOfVulnerability = makeBooleanDebuffInput({actionId: ActionId.fromSpellId(427143), fieldName: 'curseOfVulnerability', minLevel: 25});
+export const GiftOfArthas = makeBooleanDebuffInput({actionId: ActionId.fromSpellId(11374), fieldName: 'giftOfArthas', minLevel: 38});
+export const CrystalYield = makeBooleanDebuffInput({actionId: ActionId.fromSpellId(15235), fieldName: 'crystalYield', minLevel: 47});
+export const AncientCorrosivePoison = makeMultistateMultiplierDebuffInput(ActionId.fromSpellId(422996), 101, 10, 'ancientCorrosivePoison');
+
+///////////////////////////////////////////////////////////////////////////
+//                                 CONFIGS
+///////////////////////////////////////////////////////////////////////////
+
+export const RAID_BUFFS_CONFIG = [
+	// Standard buffs
+	{
+		config: AllStatsBuff,
+		picker: IconPicker,
+		stats: []
+	},
+	{
+		config: AllStatsPercentBuffAlliance,
+		picker: MultiIconPicker,
+		stats: []
+	},
+	{
+		config: AllStatsPercentBuffHorde,
+		picker: IconPicker,
+		stats: []
+	},
+	{
+		config: ArmorBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: StaminaBuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatStamina]
+	},
+	{
+		config: StrengthBuffAlliance,
+		picker: MultiIconPicker,
+		stats: [Stat.StatStrength]
```

Comment:
would do [Stat.StatStrength, Stat.StatAgility, Stat.StatAttackPower] here

Reply:
Do you know if this applies to Ranged attack power too?

Reply:
added `[Stat.StatStrength, Stat.StatAgility, Stat.StatAttackPower, Stat.StatRangedAttackPower]` but let me know if ranged AP is not needed